src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int FILE_NAME = 0;
+    private static final int RESULT_NAME = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readFile(fromFileName);
+        String result = calculateDate(data);
+        writeDateToFile(toFileName, result);
+    }
+
+    private String readFile(String fileName) {
+        StringBuilder builder = new StringBuilder();
+        File file = new File(fileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
```

Comment:
always specify as more details (filename here) in error messages as possible so it will be easier to fix in future.

Reply:
not fixed 
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/46aa0769-3e2b-480d-af9b-a7989940d8cc)